javalib/src/main/scala/java/util/logging/ErrorManager.scala

```diff
@@ -0,0 +1,25 @@
+package java.util.logging
+
+object ErrorManager {
+  val GENERIC_FAILURE = 0
+  val WRITE_FAILURE   = 1
+  val FLUSH_FAILURE   = 2
+  val CLOSE_FAILURE   = 3
+  val OPEN_FAILURE    = 4
+  val FORMAT_FAILURE  = 5
+}
+
+class ErrorManager() {
+  // The spec is a bit vague. This will implement the most
+  // obvious interpretation of outputting only once
+  private[this] var called = false
+
+  def error(msg: String, ex: Exception, code: Int): Unit = {
+    if (!called) {
+      System.err.println(msg)
+      ex.printStackTrace(System.err)
```

Comment:
`msg` and `ex` may be null. You also need to print code at the start. Look at the output from the JVM:

```scala
scala> new java.util.logging.ErrorManager().error(null, null, 42)
java.util.logging.ErrorManager: 42

scala> new java.util.logging.ErrorManager().error("abc", null, 42)
java.util.logging.ErrorManager: 42: abc

scala> new java.util.logging.ErrorManager().error("abc", new Exception, 42)
java.util.logging.ErrorManager: 42: abc
java.lang.Exception
	at $line37.$read$$iw$$iw$.<init>(<console>:11)
	at $line37.$read$$iw$$iw$.<clinit>(<console>)
	at $line37.$eval$.$print$lzycompute(<console>:7)
	at $line37.$eval$.$print(<console>:6)
	at $line37.$eval.$print(<console>)
	...
```